### PR TITLE
fix: bugfix for web3 profile

### PR DESCRIPTION
**Pull request by nuanyang233**
fix: bugfix for web3 profile

## Description

Closes #MF-1765 MF-1770

## Type of change

- [ ] Documentation
- [ ] Code refactoring (Restructuring existing code w/o changing its observable behavior)
- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (a fix or feature that would make something no longer possible to do/require old user must upgrade their Mask Network to this new version)

## Checklist

- [ ] My code follows the style guidelines of this project.
- [ ] I have performed a self-review of my own code.
  - [ ] I have removed all in development `console.log`s
  - [ ] I have removed all commented code.
- [ ] I have commented on my code, particularly in hard-to-understand areas.
- [ ] I have read [Internationalization Guide](https://github.com/DimensionDev/Maskbook/blob/develop/docs/i18n-guide.md) and moved text fields to the i18n JSON file.

If this PR depends on external APIs:

- [ ] I have configured those APIs with CORS headers to let extension requests get passed. 
  - chrome extension: `chrome-extension://[id]`
  - firefox extension: `moz-extension://[id]`
- [ ] I have delegated all web requests to the background service via the internal RPC bridge.

### CODE REVIEW

File: packages/web3-providers/src/NextID/proof.ts [hunk 1]
```diff
@@ -112,13 +112,13 @@ export class NextIDProofAPI implements NextIDBaseAPI.Proof {
     }
     async queryAllExistedBindingsByPlatform(platform: NextIDPlatform, identity: string) {
         const nextIDPersonaBindings: NextIDPersonaBindings[] = []
-        let page = 0
+        let page = 1
```
Comment (guanbinrui) [inline packages/web3-providers/src/NextID/proof.ts:115]: Does this mean the service change the pagination index?

File: packages/plugins/Web3Profile/src/SNSAdaptor/utils.ts [hunk 2]
```diff
@@ -186,7 +186,7 @@ export const getNFTList = async (walletList: WalletTypes[]) => {
                 return {
                     address,
                     collections: result?.data?.map((asset) => ({
-                        key: `${asset?.contract?.address}+${asset.tokenId}`,
+                        key: `${asset?.contract?.address}_${asset.tokenId}`,
```
Comment (guanbinrui) [inline packages/plugins/Web3Profile/src/SNSAdaptor/utils.ts:189]: Any difference?

File: packages/mask/src/utils/utils.ts [hunk 3]
```diff
@@ -65,6 +65,6 @@ export const sortPersonaBindings = (a: NextIDPersonaBindings, b: NextIDPersonaBi
     const p_b = first(b.proofs.filter((x) => x.identity === userId.toLowerCase()))
 
     if (!p_a || !p_b) return 0
-    if (p_a.created_at > p_b.created_at) return -1
+    if (p_a.last_checked_at > p_b.last_checked_at) return -1
```
Comment (guanbinrui) [inline packages/mask/src/utils/utils.ts:68]: Not only here, that we sort personas in this way. So please also update other places.

File: packages/web3-providers/src/NextID/proof.ts (hunk 1 above)
Comment (nuanyang233) [reply packages/web3-providers/src/NextID/proof.ts:115]: https://github.com/nextdotid/proof-server/blob/0065ed83f8cb9def1750718272aac5472918f0a8/docs/api.apib#L166

File: packages/plugins/Web3Profile/src/SNSAdaptor/utils.ts (hunk 2 above)
Comment (nuanyang233) [reply packages/plugins/Web3Profile/src/SNSAdaptor/utils.ts:189]: The key of all nft messages in the nft wall is split by _, we need to change it to the same format.

File: packages/web3-providers/src/NextID/proof.ts (hunk 1 above)
Comment (guanbinrui) [reply packages/web3-providers/src/NextID/proof.ts:115]: CC @lelenei So this bug existed before.

**Review by guanbinrui — APPROVED** [review]

**Pull request merged**